integration-tests/run-local-tests.sh

```diff
@@ -8,6 +8,7 @@ usage() {
     echo 'Options:'
     echo '  -t $TAGS            Tags to use for selecting scenario'
     echo '  -p                  To be set if you want to run the tests in parallel mode'
+    echo '  -k                  Keep created data, aka do not perform cleanup. Mostly used for local test run and debug.'
```

Comment:
Is it related to the PR?

Reply:
not directly but I used it a lot as it is useful for local debug :)